src/adapter/src/catalog/open.rs

```diff
@@ -19,7 +19,7 @@ use tracing::{info, warn};
 use uuid::Uuid;
 
 use mz_catalog::builtin::{
-    Builtin, Fingerprint, BUILTINS, BUILTIN_CLUSTERS, BUILTIN_PREFIXES, BUILTIN_ROLES,
```

Comment:
nit: At first glance it was not clear to me what `Sensitivity` meant here, later I figured it was if the information is sensitive. It would be great if the name captured the intention of "who can view this data", anything like VisibilityControl/AccessControl etc. or even DataSensitivity to make it more descriptive.